### PR TITLE
Update zga to 0.0.9 with dependencies fix

### PR DESCRIPTION
**Pull request by laxeye**
Update zga: 0.0.8 → 0.0.9

Added minimap2 requirement that missed in auto-bump commit.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by laxeye** [comment]
@BiocondaBot please add label

File: recipes/zga/meta.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@ requirements:
     - dfast
     - nxtrim
     - racon
+    - minimap2
```
Comment (bgruening) [inline recipes/zga/meta.yaml:39]: you need to bump the build number as well

Comment (laxeye) [reply recipes/zga/meta.yaml:39]: > you need to bump the build number as well
Dear Björn,
should i make new PR with this change?

Comment (bgruening) [reply recipes/zga/meta.yaml:39]: No, you can change this PR.

Comment (laxeye) [reply recipes/zga/meta.yaml:39]: > No, you can change this PR.

I did it, but now I got an error:
ERROR: recipes/zga/meta.yaml:14: build_number_needs_reset: The recipe build number should be reset to 0

**Pull request closed without merging**